src/docs/asciidoc/link-attributes.adoc

```diff
@@ -1,8 +1,9 @@
-:assertj-core-javadoc-root:              https://www.javadoc.io/doc/org.assertj/assertj-core/latest/
-:assertj-guava-javadoc-root:             https://www.javadoc.io/doc/org.assertj/assertj-guava/latest/
+:assertj-core-javadoc-root:              https://www.javadoc.io/doc/org.assertj/assertj-core/latest/org.assertj.core/
+:assertj-guava-javadoc-root:             https://www.javadoc.io/doc/org.assertj/assertj-guava/latest/org.assertj.guava/
```

Comment:
I think you've spotted an important issue: these variables target `latest`, whereas we should use `assertj-version` consistently, which is already done in a few cases, like the first ones you changed.

Specifically, we are now publishing milestones for version 4, but we don't want the user guide to point to them yet.

Would you mind adjusting the PR in this direction?